### PR TITLE
pypiwin32 220 sucks, install 221 manually

### PR DESCRIPTION
**Pull request by chuaaron**
boy, Windows really took my time to make it work!

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -149,6 +149,10 @@ On Windows::
 (If that failed, try::
     
     $ pip install pypiwin32
+ 
+ then grab a proper version of pypiwin32 installer and install it::
+ 
+    https://github.com/mhammond/pywin32/releases
```
Comment (gregmalcolm) [inline README.rst]: Yeah, let's completely get rid of references to pypiwin32 and simplify. How about something like this?

`On Windows:

Install pywin32 from here:

https://github.com/mhammond/pywin32/releases`

**Comment by kjc** [comment]
I don't own a Windows box, so I have no idea what to do with this PR.

**Comment by gregmalcolm** [comment]
I might need to write a fresh PR to handle this. It needs simplifying a little.

**Comment by gregmalcolm** [comment]
The update has gone through. Sorry to keep it waiting so long!

**Pull request merged**